mava/components/jax/building/system_init.py

```diff
@@ -131,11 +135,10 @@ def config_class() -> Optional[Callable]:
 
 
 @dataclass
-class RandomSamplingSystemInitConfig:
+class RandomSamplingSystemInitConfig(BaseSystemInitConfig):
```

Comment:
As discussed, this one should please not inherit from `BaseSystemInitConfig`, since it can never have shared weights.

Reply:
Updated :+1: